manifests/charts/base/templates/crds.yaml

```diff
@@ -1,4 +1,8 @@
 {{- if .Values.base.enableCRDTemplates }}
 {{ .Files.Get "crds/crd-all.gen.yaml" }}
 {{ .Files.Get "crds/crd-operator.yaml" }}
+{{ .Files.Get "crds/maistra.io_meshfederations.yaml" }}
+{{ .Files.Get "crds/maistra.io_federationstatuses.yaml" }}
+{{ .Files.Get "crds/maistra.io_serviceexports.yaml" }}
+{{ .Files.Get "crds/maistra.io_serviceimports.yaml" }}
```

Comment:
None of these files exist. Shouldn't they be replaced with the files below?
```
{{ .Files.Get "crds/federation.io_exportedservicesets.yaml" }}
{{ .Files.Get "crds/federation.io_importedservicesets.yaml" }}
{{ .Files.Get "crds/federation.io_servicemeshpeers.yaml" }}
```

Reply:
Why the tests are passing then? oO

Reply:
I'm guessing that the logic that installs maistra CRDs for integration tests covers this? Good catch Jacek

Reply:
Anyway, good catch. Thanks. That's "fixed" now in 2c52f2e71f